dashboard/src/components/ErrorAlert/MessageAlert.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { shallow } from "enzyme";
+import * as React from "react";
+
+import ErrorPageHeader from "./ErrorAlertHeader";
+import MessageAlert from "./MessageAlert";
+
+it("renders the heading passed to it", () => {
+  const wrapper = shallow(
+    <MessageAlert>
+      <div>test</div>
+    </MessageAlert>,
+  );
+  expect(wrapper.text()).toContain("test");
+  expect(wrapper.find(".message__content").text()).toContain("test");
```

Comment:
should we add some snapshots here so we can tell if things have structurally changed? I think they can be useful as indicators of changes, and also can be easier for writing simple tests like this where you're just checking content.

Reply:
I'd like to raise a discussion for that. I don't find snapshots very helpful because it doesn't allow you to express intentionality. How would you say that the important part of the snapshot is that it renders a element with the class `message_content` with the text `test`?

Reply:
I agree with that, I think snapshots are useful **in addition** to more specific assertions as they can catch unexpected structural changes.